Update UI design of detail page

alecslupu opened the pull request:
#### :tophat: What? Why?
This PR changes the Proposal page in order to apply the changes requested in #13328

- [x] Move the status cell above the author, reduce the gap from 2rem to 1 rem b56237f9aa81554fbf0639f8865af235e7117a41
- [x] Move Tags above the actions effaa29468
- [x] Move the Share button 5d0e8c8b7d
- [x] Add kebab menu 3d4ac4fac7
- [x] Move follower list 100a2b051c
- [x] Expand followers list  f93675faa4

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #13328

#### Testing
*Describe the best way to test or validate your PR.*

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!

alecslupu commented:
As per this PR implementation implementation, several changes have not been implemented, and require further clarification: 
- [ ] Endorser's pop-up is not sorting followers based on relation with current user
- [ ] Endorser's pop-up is displaying a svg (to respect design with all other pop-ups)
- [ ] Endorser's pop-up is not showing a horizontal rule
- [ ] Endorser's pop-up is not full screen
- [ ] Endorser's pop-up - The follow user does not have 2 states  (transparent for follow, and full for not followed), implementing this will change the behavior in all the places. It can be implemented as a stand alone class, but needs to be confirmed
- [ ] Endorser's pop-up - The follow user button has a wrong caption.
- [ ] Endorser's pop-up - The follow user button has a svg.  
- [ ] Between like and comments is no line
- [ ] There is no "show more" button 
- [ ] In mobile version the space after endorsers list is too big (need to search for alternatives)
- [ ] In the Kebab menu the svg and text are inverted (as this is the order on the entire app)
- [ ] Not all the fonts and texts have been verified

![image](https://github.com/user-attachments/assets/cb161eff-5907-4ccf-9dad-f744f12d827e)
![image](https://github.com/user-attachments/assets/db494682-039e-422a-a636-fa8da8bbe920)
![image](https://github.com/user-attachments/assets/e5daa3f7-7082-4eab-82d9-c10be9af30c7)
![image](https://github.com/user-attachments/assets/152a6e57-d5a6-4f80-a5c3-0866c840550d)
![image](https://github.com/user-attachments/assets/0f236c94-5ff5-4b14-92d3-8a91ae20eb0a)

carolromero reviewed (changes requested):
@alecslupu I've reviewed this and overall it's working great! Here's some feedback from the review:

- [ ] The 'Like' button behaviour needs to be adjusted like in this flow: https://www.figma.com/proto/ixNrQfqsSMCzpPYyUdefOt/Secondary-actions?node-id=251-29338&node-type=canvas&t=k2ASFpE72R5lFN1z-9&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1&starting-point-node-id=251%3A29338&show-proto-sidebar=1
- [ ] When clicking on the “Comment” button it should scroll to the text box to make a new comment.
- [ ] Kebab menu must incorporate author actions 'Edit' and 'Withdraw'.

alecslupu commented:
Close debate: 
- moved user edit debate 
- moved admin edit debate
- user closes debate 
- admin closes debate
![image](https://github.com/user-attachments/assets/8108ff2c-0860-4035-983a-f76f8758d8d8)

Meeting :
- Moved user edit meeting 
- Moved user withdraws meeting 
- Moved user closes meeting 

![image](https://github.com/user-attachments/assets/94c5c2cc-d68e-4e8b-a6de-a91e53f086b8)
![image](https://github.com/user-attachments/assets/53d2cffd-55c8-4419-84d9-a2a59a6e5c7d)

Proposals : 
![image](https://github.com/user-attachments/assets/c02cc12b-18a5-461d-b87b-dd7deed514ed)
![image](https://github.com/user-attachments/assets/70589c3c-70d5-4380-a8df-45f847e38fc7)
![image](https://github.com/user-attachments/assets/633c1009-6489-40db-8f9c-e98ad79263ba)

Collaborative drafts
![image](https://github.com/user-attachments/assets/cc33592b-276f-4375-86e3-1f23217f8f52)

andreslucena reviewed (approved):
Checked it out locally and the actions that I could find were already migrated to the kebab menu. LGTM!
